misoc/software/libbase/crt0-or1k.S

```diff
@@ -19,7 +19,18 @@
 
 #include <spr-defs.h>
 
-#define EXCEPTION_STACK_SIZE (4*32)
+/*
+ * OR1K Architecture has a 128 byte "red zone" after the stack that can not be
+ * touched by exception handlers. GCC uses this red zone for locals and
```

Comment:
Not just GCC, any ABI-compliant compiler (e.g. Clang) can use it.

Reply:
Should I change this to `The red zone is generally used for ......`?

Reply:
"The red zone is used for". That's its only purpose.

Reply:
Well, technically you could use it for anything your heart desires... Maybe I should just remove the `The red zone is used for` bit and just have a link to the wikipedia article at https://en.wikipedia.org/wiki/Red_zone_(computing) ?